autoload/scriptease.vim

```diff
@@ -308,7 +308,11 @@ endfunction
 
 " Section: :Messages
 
-function! scriptease#messages_command(bang) abort
+function! scriptease#messages_command(bang, ...) abort
```

Comment:
Use a named parameter like `arg` rather than `...`. `<q-args>` will be `''` for no argument, so there will always be exactly one parameter.

Reply:
done!